c7n/query.py

```diff
@@ -273,6 +273,65 @@ def get_query(self):
             self.manager.session_factory, self.manager)
 
 
+@sources.register('cloud-control')
+class CloudControl:
+
+    resources_expr = jmespath.compile('ResourceDescriptions[].Properties')
+
+    def __init__(self, manager):
+        self.manager = manager
+
+    def get_permissions(self):
+        # cfn type registry implementations use undefined permission
```

Comment:
this isn't true, they are published as part of a json schema cfn specific version.

Reply:
albeit its not universal in application/definition.. re handler section